src/BenchmarkDotNet/Toolchains/Roslyn/RoslynToolchain.cs

```diff
@@ -14,6 +15,12 @@ public class RoslynToolchain : Toolchain
     {
         public static IToolchain Instance = new RoslynToolchain();
 
+        [PublicAPI]
+        public RoslynToolchain (MonoRuntime mono) : base("Roslyn", new Generator(), new MonoAotBuilder(mono), new Executor())
```

Comment:
nit: spacing
important: name

```suggestion
        public RoslynToolchain(MonoRuntime mono) : base("MonoAOT", new Generator(), new MonoAotBuilder(mono), new Executor())
```